<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::MetaDataDialog</class>
 <widget class="QDialog" name="Otter::MetaDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Otter::LineEditWidget" name="titleLineEditWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Otter::LineEditWidget" name="descriptionLineEditWidget"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>Version:</string>
       </property>
       <property name="buddy">
        <cstring>versionLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Otter::LineEditWidget" name="versionLineEditWidget"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author:</string>
       </property>
       <property name="buddy">
        <cstring>authorLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Otter::LineEditWidget" name="authorLineEditWidget"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="homepageLabel">
       <property name="text">
        <string>Homepage:</string>
       </property>
       <property name="buddy">
        <cstring>homepageLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Otter::LineEditWidget" name="homepageLineEditWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::MetaDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::MetaDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
